gpx.cc

```diff
@@ -1887,13 +1904,14 @@ gpx_write()
 static void
 gpx_free_gpx_global()
 {
-  gpx_rm_from_global(&gpx_global->name);
-  gpx_rm_from_global(&gpx_global->desc);
-  gpx_rm_from_global(&gpx_global->author);
-  gpx_rm_from_global(&gpx_global->email);
-  gpx_rm_from_global(&gpx_global->url);
-  gpx_rm_from_global(&gpx_global->urlname);
-  gpx_rm_from_global(&gpx_global->keywords);
+  delete gpx_global->name;
```

Comment:
We could make gpx_global a class and then all the containers inside it would delete when gpx_global is deleted. One less xmalloc/xfree and better future safety.